frontend/src/routes/issues/table/+layout.svelte

```diff
@@ -108,55 +108,17 @@
 	}
 
 	async function handleDownloadCsv() {
-		const allServiceRequests = await libre311.getAllServiceRequests({});
+		let searchParams = $page.url.searchParams.toString();
 
-		// Sanatize Requests
-		for (let request of allServiceRequests) {
-			request.address = request.address.replace(/,/g, '');
-		}
+		// Remove `page` and `page_size`
+		searchParams = searchParams.replace(/page_size=\d+&page=\d+&/g, '');
 
-		const csvContent = convertToCSV(allServiceRequests);
+		const allServiceRequests = await libre311.downloadServiceRequests(searchParams);
 
-		const blob = new Blob([csvContent], { type: 'text/csv;charset=utf-8' });
+		const blob = new Blob([allServiceRequests], { type: 'text/csv;charset=utf-8' });
 		saveAs(blob, 'service-requests.csv');
```

Comment:
Did you try
`<a href=<the-link>  download />` ? 
We shouldn't need to call the blob constructor and the saveAs method.

Reply:
Let me try that.  This way I would keep the download apart of the service instead of the component